src/schema.py

```diff
@@ -141,12 +142,13 @@ def resolve_account_info(self, info, session_id=None):
         continue
       dt_utc = parser.parse(t['actualDate']).astimezone(pytz.timezone('UTC'))
       dt_est = dt_utc.astimezone(pytz.timezone('US/Eastern'))
+      location = t['locationName'][:t['locationName'].rfind(' ')]
```

Comment:
`location = t['locationName'].rsplit(' ', 1)[0]` is probably cleaner, but check that the code works

Reply:
ohh so fresh and so cleeeaaan